src/components/Navbar.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { StyleSheet, View, Image } from 'react-native';
+import colors from '../assets/colors.js'
+import images from '../assets/images.js'
+
+class Navbar extends React.Component {
+  render() {
+    return (
+      <View style={styles.navbar}>
+        <View style={styles.imageContainer}>
+          <Image source={images.magGlass}/>
+        </View>
+        <View style={styles.imageContainer}>
+          <Image source={images.camera}/>
+        </View>
+        <View style={styles.imageContainer}>
+          <Image source={images.recycleBook}/>
+        </View>
+        <View style={styles.imageContainer}>
+          <Image source={images.gear}/>
+        </View>
+      </View>
+    );
+  }
+}
+
+const styles = StyleSheet.create({
+  navbar: {
+    position: 'absolute', 
```

Comment:
I defined a metric that takes care of this. It would be used by putting the content of each route into a View and assigning a StyleSheet that includes: `marginBottom: metrics.navbarMargin`